javascript-timers/interval.js

```diff
@@ -14,9 +14,7 @@ function text() {
     var count = parseInt(text.textContent);
     --count;
     text.textContent = String(count);
-    return;
-  }
-  if (text.textContent === '1') {
+  } else if (text.textContent === '1') {
```

Comment:
It can just be `else`. If the textContent is not `!== "1"`, then it is `=== "1"`, there is no alternative